include/psa/crypto_platform.h

```diff
@@ -83,7 +83,7 @@ static inline int mbedtls_key_owner_id_equal(mbedtls_key_owner_id_t id1,
  */
 #if defined(MBEDTLS_PSA_CRYPTO_SPM)
 #define PSA_CRYPTO_SECURE 1
-#include "crypto_spe.h"
+#include "../tests/include/spe/crypto_spe.h"
```

Comment:
No, [that change is wrong](https://github.com/Mbed-TLS/mbedtls/issues/8259). Production code cannot reference test paths.